src/site/SearchController.php

```diff
@@ -0,0 +1,71 @@
+<?hh // strict
+
+use HHVM\UserDocumentation\GuidesIndex;
+use HHVM\UserDocumentation\APIIndex;
+use HHVM\UserDocumentation\SearchResultSet;
+
+final class SearchController extends WebPageController {
+  private string $term;
+  private bool $term_set;
+
+  public function __construct(ImmMap<string, string> $params) {
+    parent::__construct($params);
+    $this->term = "";
+    $this->term_set = false;
```

Comment:
- this constructor is no longer valid - hh_client should complain
- not actually needed - could use static initializers instead
- why $term_set = false instead of ?string $term = null